<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite tests="9" failures="8" time="0.630" name="./failing-tests.sh">
		<properties>
			<property name="go.version" value="go1.6.3"></property>
		</properties>
		<testcase classname="failing-tests.sh" name="it_should_support_defer_even_on_fails" time="0.020">
			<failure message="Failed" type="">+ it_should_support_defer_even_on_fails&#xA;+ echo hallo&#xA;+ defer rm out&#xA;+ __DEFERRED_CALLS=&#39;rm out; &#39;&#xA;+ assert &#39;test -e out&#39;&#xA;+ &#39;[&#39; -z &#39;&#39; &#39;]&#39;&#xA;+ test -e out&#xA;+ got=0&#xA;+ expected=0&#xA;+ description=&#39;expecting command &#39;\&#39;&#39;test -e out&#39;\&#39;&#39; to succeed&#39;&#xA;+ set +x&#xA;######################################## PASSED TEST: expecting command &#39;test -e out&#39; to succeed (&#39;0&#39; == &#39;0&#39;)&#xA;+ assert 0 1&#xA;+ &#39;[&#39; -z 1 &#39;]&#39;&#xA;+ got=0&#xA;+ expected=1&#xA;+ description=&#xA;+ set +x&#xA;failed expectation: &#39;0&#39; == &#39;1&#39; &#xA;######################################## FAILED TEST: &#39;0&#39; == &#39;1&#39;&#xA;%%%%%%%%%%%%%%%%%%%% executing defer commands: &#39;eval &#34;rm out; &#34; &amp;&gt; /tmp/.spec.sh.deferred_calls.VKCFSG&#39;&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:11:it_should_support_defer_even_on_fails</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_execute_command" time="0.000">
			<failure message="Failed" type="">+ it_should_execute_command&#xA;+ assert &#39;ls /xxxtmp&#39;&#xA;+ &#39;[&#39; -z &#39;&#39; &#39;]&#39;&#xA;+ ls /xxxtmp&#xA;ls: /xxxtmp: No such file or directory&#xA;+ got=1&#xA;+ expected=0&#xA;+ description=&#39;expecting command &#39;\&#39;&#39;ls /xxxtmp&#39;\&#39;&#39; to succeed&#39;&#xA;+ set +x&#xA;failed expectation: expecting command &#39;ls /xxxtmp&#39; to succeed (&#39;1&#39; == &#39;0&#39;) &#xA;######################################## FAILED TEST: expecting command &#39;ls /xxxtmp&#39; to succeed (&#39;1&#39; == &#39;0&#39;)&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:20:it_should_execute_command</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_reflect_exit_code" time="0.000">
			<failure message="Failed" type="">+ it_should_reflect_exit_code&#xA;+ return 3&#xA;+ res=3&#xA;+ set +x&#xA;error code: 3&#xA;error occured in ./failing-tests.sh:25:it_should_reflect_exit_code</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_match_exit_code" time="0.000">
			<failure message="Failed" type="">+ it_should_match_exit_code&#xA;+ false&#xA;+ assert 0 0&#xA;+ &#39;[&#39; -z 0 &#39;]&#39;&#xA;+ got=0&#xA;+ expected=0&#xA;+ description=&#xA;+ set +x&#xA;######################################## PASSED TEST: &#39;0&#39; == &#39;0&#39;&#xA;+ assert 0 0 &#39;0 should be 0&#39;&#xA;+ &#39;[&#39; -z 0 &#39;]&#39;&#xA;+ got=0&#xA;+ expected=0&#xA;+ description=&#39;0 should be 0&#39;&#xA;+ set +x&#xA;######################################## PASSED TEST: 0 should be 0 (&#39;0&#39; == &#39;0&#39;)&#xA;+ assert 0 2&#xA;+ &#39;[&#39; -z 2 &#39;]&#39;&#xA;+ got=0&#xA;+ expected=2&#xA;+ description=&#xA;+ set +x&#xA;failed expectation: &#39;0&#39; == &#39;2&#39; &#xA;######################################## FAILED TEST: &#39;0&#39; == &#39;2&#39;&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:29:it_should_match_exit_code</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_match_string" time="0.000">
			<failure message="Failed" type="">+ it_should_match_string&#xA;++ echo hallo&#xA;++ tr l x&#xA;+ assert haxxo haggo&#xA;+ &#39;[&#39; -z haggo &#39;]&#39;&#xA;+ got=haxxo&#xA;+ expected=haggo&#xA;+ description=&#xA;+ set +x&#xA;failed expectation: &#39;haxxo&#39; == &#39;haggo&#39; &#xA;######################################## FAILED TEST: &#39;haxxo&#39; == &#39;haggo&#39;&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:38:it_should_match_string</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_match_string_with_description" time="0.000">
			<failure message="Failed" type="">+ it_should_match_string_with_description&#xA;++ echo hallo&#xA;++ tr l x&#xA;+ assert haxxo haggo &#39;transforming string should work&#39;&#xA;+ &#39;[&#39; -z haggo &#39;]&#39;&#xA;+ got=haxxo&#xA;+ expected=haggo&#xA;+ description=&#39;transforming string should work&#39;&#xA;+ set +x&#xA;failed expectation: transforming string should work (&#39;haxxo&#39; == &#39;haggo&#39;) &#xA;######################################## FAILED TEST: transforming string should work (&#39;haxxo&#39; == &#39;haggo&#39;)&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:43:it_should_match_string_with_description</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_match_regexp" time="0.310">
			<failure message="Failed" type="">+ it_should_match_regexp&#xA;+ sleep 0.3&#xA;+ assert_match aaa:88X09 &#39;aaa:[0-9]{5}&#39;&#xA;+ set +o pipefail&#xA;+ printf aaa:88X09&#xA;+ grep -E -m1 -o &#39;aaa:[0-9]{5}&#39;&#xA;+ head -n1&#xA;+ grep -E &#39;aaa:[0-9]{5}&#39;&#xA;+ assert 1 0 &#39;checking &#39;\&#39;&#39;aaa:88X09&#39;\&#39;&#39; to match /aaa:[0-9]{5}/&#39;&#xA;+ &#39;[&#39; -z 0 &#39;]&#39;&#xA;+ got=1&#xA;+ expected=0&#xA;+ description=&#39;checking &#39;\&#39;&#39;aaa:88X09&#39;\&#39;&#39; to match /aaa:[0-9]{5}/&#39;&#xA;+ set +x&#xA;failed expectation: checking &#39;aaa:88X09&#39; to match /aaa:[0-9]{5}/ (&#39;1&#39; == &#39;0&#39;) &#xA;######################################## FAILED TEST: checking &#39;aaa:88X09&#39; to match /aaa:[0-9]{5}/ (&#39;1&#39; == &#39;0&#39;)&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:48:it_should_match_regexp</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="it_should_have_seven_failing_tests" time="0.000">
			<failure message="Failed" type="">+ it_should_have_seven_failing_tests&#xA;+ assert 1 2&#xA;+ &#39;[&#39; -z 2 &#39;]&#39;&#xA;+ got=1&#xA;+ expected=2&#xA;+ description=&#xA;+ set +x&#xA;failed expectation: &#39;1&#39; == &#39;2&#39; &#xA;######################################## FAILED TEST: &#39;1&#39; == &#39;2&#39;&#xA;+ exit 1&#xA;error code: 1&#xA;error occured in ./failing-tests.sh:54:it_should_have_seven_failing_tests</failure>
		</testcase>
		<testcase classname="failing-tests.sh" name="after_all" time="0.000"></testcase>
	</testsuite>
</testsuites>
